contracts/ShortsReelsVideos.sol

```diff
@@ -61,28 +82,88 @@ contract ShortsReelsVideos is
         maintainer = _maintainer;
     }
 
-    function payForVideoVanityURLAccess(address _user, string memory _name, string memory _aliasName, uint256 _paidAt) external onlyMaintainer {
+    function payForContentAccess(address _user, string memory _name, string memory _aliasName, uint256 _paidAt) public payable {
         bytes32 tokenId = keccak256(bytes(_name));
         require(
-            !checkVideoVanityURLAccess(_user, _name, _aliasName),
+            !checkContentAccess(_user, _name, _aliasName),
             "VanityURL: already paid"
         );
 
         // check the payment timestamp
         require(_paidAt <= block.timestamp, "VanityURL: invalid time");
 
+        require(contentRecords[tokenId][_aliasName].owner != address(0x0), "Content not created");
```

Comment:
Same. I think we can remove the owner check.
FYI, here are the parameter examples.
"_name" - Alice.1.country
"_aliasName" - "/videos", "/videos/1", ...

"_aliasName" depends on the "_name", so the above examples represent these vanity urls.
"Alice.1.country/videos", "Alice.1.country/videos/1"

And these vanity urls are managed by VanityURL contract. (create, read, update, delete)
https://github.com/harmony-one/.1.country/blob/v1.1/contracts/contracts/VanityURL.sol#L96
https://github.com/harmony-one/.1.country/blob/v1.1/contracts/contracts/VanityURL.sol#L127
https://github.com/harmony-one/.1.country/blob/v1.1/contracts/contracts/VanityURL.sol#L143

Maybe setting the price can be moved to VanityURL contract and here, we can only handle the payment check.